src/main/java/ladder/domain/user/User.java

```diff
@@ -0,0 +1,23 @@
+package ladder.domain.user;
+
+public class User {
+	private final String name;
+
+	public User(String name) {
+		this.name = name;
+	}
+
+	public static User createUser(String name) {
+		if(name == null || name.isEmpty()){
+			throw new IllegalArgumentException("이름은 null이나 빈 문자열이 될 수 없습니다.");
+		}
+		if(name.length() > 5){
```

Comment:
```suggestion
		if (name.length() > 5) {
```
`5`는 매직넘버이니 추출되면 좋을 것 같습니다 🙇 
그래야 아래 `Exception`의 문구도 추출된 상수로 한번에 변경이 가능할 테니까요 😄